README.adoc

```diff
@@ -1,148 +1,16 @@
 = KEEP & tBTC local setup
 
-== Setting up development environment
-If you’re on macOS, install Homebrew and https://github.com/keep-network/keep-core/blob/master/scripts/macos-setup.sh[macos-setup.sh] script to setup all necessary tools.
+== Installation
 
-If you are not on macOS or you don't have Homebrew, you need to install:
+This repository can be used in two different ways described below:
```

Comment:
We should describe what this repository is for and what components are installed. As a newcomer, I'd have no idea what's here. We should mention tBTC, tBTC dApp, ECDSA client, core client, Electrum, Bitcoin, and Ethereum node, and whatever I missed here followed by links to the repositories. We should also say it contains and allows for execution of end-to-end automated system tests for tBTC.

Reply:
Those details live in linked docs (`manual-setup` and `auto-setup`) but agree. A small introduction here will be a good starting point.